pkg/db/transaction.go

```diff
@@ -153,6 +154,7 @@ func (t *TxConnection) Query(query string, args ...interface{}) (DataRows, error
 }
 
 func (t *TxConnection) Exec(query string, args ...interface{}) (sql.Result, error) {
+	t.logger.Debugf("Transaction Exec(): %s | %v", query, args)
```

Comment:
Do we want to add this also for the other statements or as opt-in on the calling service layer (e.g. Repository)? I feel like only having this on exec seems a bit arbitrary

Reply:
In my opinion, this was used for debugging and can be removed in a follow-up PR.